syntax = "proto3";

message LocationMessage {
  int32 person_id = 1;
  float longitude = 2;
  float latitude = 3;
  string creation_time = 4;
}

message Empty {

}

service LocationService {
    rpc Create(LocationMessage) returns (Empty);
}